app/schemas/federation_entities.json

```diff
@@ -0,0 +1,366 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "id": "https://diasporafoundation.org/schemas/federation_entities-1.0.0.json",
```

Comment:
Yes, I think that's the best solution for now (and we can always separate it later). Sorry for the back and forth here, but that just came to my mind that it probably makes the development much more complicated when we split it (but still need to update it every time).

And I also would keep the version together for now, so first release with this would be 0.2.0, and if then 0.2.1 maybe only is a bugfix release without entity changes, so I'd simply keep the schema version at 0.2.0, and if in 0.2.2 we add a property somewhere then I'd move the schema version to 0.2.2. I think that is OK to have gaps between schema versions here (maybe we can add a list with schema versions and schema changelog somewhere?).